From: Bjorn Andersson <bjorn.andersson@linaro.org>
Date: Sun, 5 Feb 2023 17:28:07 +0200
Subject: dt-bindings: power: supply: Add Lenovo Yoga C630 EC

Add binding for the Embedded Controller found in the Qualcomm
Snapdragon-based Lenovo Yoga C630.

Signed-off-by: Bjorn Andersson <bjorn.andersson@linaro.org>
Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Reviewed-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
---
 .../bindings/{platform => power/supply}/lenovo,yoga-c630-ec.yaml      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 copy Documentation/devicetree/bindings/{platform => power/supply}/lenovo,yoga-c630-ec.yaml (93%)

diff --git a/Documentation/devicetree/bindings/platform/lenovo,yoga-c630-ec.yaml b/Documentation/devicetree/bindings/power/supply/lenovo,yoga-c630-ec.yaml
similarity index 93%
copy from Documentation/devicetree/bindings/platform/lenovo,yoga-c630-ec.yaml
copy to Documentation/devicetree/bindings/power/supply/lenovo,yoga-c630-ec.yaml
index 3180ce1..3797734 100644
--- a/Documentation/devicetree/bindings/platform/lenovo,yoga-c630-ec.yaml
+++ b/Documentation/devicetree/bindings/power/supply/lenovo,yoga-c630-ec.yaml
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
 %YAML 1.2
 ---
-$id: http://devicetree.org/schemas/platform/lenovo,yoga-c630-ec.yaml#
+$id: http://devicetree.org/schemas/power/supply/lenovo,yoga-c630-ec.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
 title: Lenovo Yoga C630 Embedded Controller.
 
 maintainers:
-  - Bjorn Andersson <andersson@kernel.org>
+  - Bjorn Andersson <bjorn.andersson@linaro.org>
 
 description:
   The Qualcomm Snapdragon-based Lenovo Yoga C630 has an Embedded Controller
